telegram/bot.py

```diff
@@ -222,6 +207,23 @@ def __init__(
                 private_key, password=private_key_password, backend=default_backend()
             )
 
+    # TODO: After https://youtrack.jetbrains.com/issue/PY-50952 is fixed, we can revisit this and
+    # consider adding Paramspec from typing_extensions to properly fix this. Currently a workaround
+    def log(  # type: ignore[no-untyped-def] # skipcq: PY-D0003
```

Comment:
in light of our efforts to make clear what is public and what is not: let's make this a private method :)

Reply:
I thought of doing that, but then I saw that that didn't show up in the docs, so didn't change it. Will change anyway.

Reply:
IIRC that's just because it doesn't have a docstring. auto-complete will still show it …